store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9629,4 +9629,8 @@ TODO: delete them permanently from here
     <desc>Account thumbnail photo</desc>
 </attr>
 
+<attr id="5001" name="zimbraPrefDisplayTimeInMailList" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInfo,accountInherited" since="8.8.8">
```

Comment:
Attr id 5001 has been set on snickers. Is it okay on develop branch?

Reply:
updated attr id to be 3022 to match with develop branch